packages/ng/toast/toasts.component.html

```diff
@@ -3,7 +3,11 @@
 		<span *ngIf="toast.type" class="toast-item-icon lucca-icon" aria-hidden="true" [ngClass]="iconClassByToastType[toast.type]"></span>
 		<div class="toast-item-content">
 			<div *ngIf="toast.title"><strong>{{ toast.title }}</strong></div>
+			@if (isStringPortalContent(toast.message)) {
```

Comment:
You had to do this for `innerHTML` support? (`*luPortal` accepts string but not HTML I guess)

Reply:
Yes.